bin/lib/zopen-clone

```diff
@@ -0,0 +1,229 @@
+#!/bin/sh
+# Clone a zopen instance
+
+if [ -z "$utildir" ]; then
+  export utildir="$( cd "$(dirname "$0")" >/dev/null 2>&1 && pwd -P )"
+fi
+
+. "${utildir}/common.inc"
+
+printHelp(){
+cat << HELPDOC
+zopen clone is a utility for z/OS Open Tools to generate a clone
+an existing zopen environment. For example a user can install
+to a test area and when content that the behaviour is correct, clone
+to a production area.
+
+Usage: zopen clone [OPTION] [DESTINATION]...
+
+Options:
+  -f, --from        the zopen envionrment to copy from; if not present,
+                    the default is taken from ZOPEN_ROOTFS (the current
+                    zopen environment)
+  -y, --yes         automatically answer 'yes' to prompts; existing 
+                    target filesystems will be purged before clone
+                    occurs
+  -v, --verbose     run in verbose mode
+  -h, -?, --help    display this help and exit
+
+Examples:
+  zopen init        interactively bootstrap a zopen environment
+  zopen init --releaseline-dev
+                    interactively bootstrap a zopen environment that
+                    will use Development Releaseline packages
+  zopen --clone /prod
+                    clone the currently active zopen environment to
+                    the '/prod' location
+  zopen --yes --append-to-profile --fs-layout fhs /zopen
+                    non-interactively create a zopen environment at 
+                    location '/zopen' on disk, with packages installed
+                    to '/zopen/opt'. The user's .profile will be
+                    updated to source the configuration file at
+                    '/zopen/etc/zopen-config' when new terminal 
+                    sessions start
+
+Report bugs at https://github.com/ZOSOpenTools/meta/issues 
+
+HELPDOC
+}
+
+args=$*
+
+printHeader "Cloning zopen environment"
+
+verbose=false
+debug=false
+yesToPrompts=false
+while [ $# -gt 0 ]; do
+  case "$1" in
+    "-f" | "--from")
+      shift
+      [ $# -lt 1 ] && printError "Missing 'from' directory"    
+      fromDir="$1"
+      ;;
+    "-h" | "--help" | "-?")
+      printHelp "${args}"
+      exit 0
+      ;;
+    "-v" | "--verbose")
+      verbose=true
+      ;;
+    "--debug") 
+      verbose=true
+      debug=true
+      ;;
+    "--xdebug")
+      set -x
+      ;;
+    "--yes" | "-y")
```

Comment:
Can you add a —version in?
Note I moved stuff around so the new zopen-clone will need to be in bin. 
for consistency I have a short function at the start of all zopen scripts - ca you add that in?

Reply:
done